modules/core/04-channel/keeper/upgrade_test.go

```diff
@@ -1150,11 +1151,25 @@ func (suite *KeeperTestSuite) TestWriteUpgradeOpenChannel() {
 			suite.Require().NoError(path.EndpointA.ChanUpgradeAck())
 			suite.Require().NoError(path.EndpointB.ChanUpgradeConfirm())
 
-			// Ack packet to delete packet commitment before calling WriteUpgradeOpenChannel
+			// Ack packets to delete packet commitments before calling WriteUpgradeOpenChannel
 			err = path.EndpointA.AcknowledgePacket(packet, ibctesting.MockAcknowledgement)
 			suite.Require().NoError(err)
 
+			err = path.EndpointB.AcknowledgePacket(packet0, ibctesting.MockAcknowledgement)
+			suite.Require().NoError(err)
+
 			ctx := suite.chainA.GetContext()
+
+			// assert that sequence (reset in ChanUpgradeTry) is 1 because channel is UNORDERED
```

Comment:
I don't understand the reset comment

Reply:
Should we also be assert the counterpartyNextSequenceSend is 2 here?

Reply:
i believe we set the sequences to 1 so that the upgrade proof can be verified without the exact last sequence send